### PR TITLE
Radioactive Turfs (New radiation system) ツ

### PR DESCRIPTION
**Pull request by xxalpha**
i am a meme

### CODE REVIEW

**Comment by YotaXP** [comment]
Just what we needed, singulo damage being even more unrepairable.

**Comment by xxalpha** [comment]
Irrelevant, once it's out you can do very little to stop it.

**Comment by Ultimate-Chimera** [comment]
:+1:

File: code/modules/research/experimentor.dm [hunk 1]
```diff
@@ -283,7 +283,7 @@
 			ejectItem()
 		if(prob(EFFECT_PROB_VERYLOW-badThingCoeff))
 			visible_message("<span class='danger'>[src] malfunctions, melting [exp_on] and leaking radiation!</span>")
-			radiation_pulse(get_turf(src), 1, 1, 25, 1)
+			irradiate(7, TRUE, name)
```
Comment (GunHog) [inline code/modules/research/experimentor.dm]: Double this!

**Comment by PKPenguin321** [comment]
rip lag removal

**Comment by Anonmare** [comment]
"We removed ALL the lag!"
"Let's add something incredibly laggy!"

Still :+1: though

**Comment by Core0verload** [comment]
I simply don't like it at all. :-1:

**Comment by Sestren413** [comment]
Interesting. Have you altered the radiation random event to work with the new system?

**Comment by Thunder12345** [comment]
:-1: Not how radiation works
Just being in the presence of a radiation emitting singularity will do exactly what the current system does, and irradiate everything nearby. The area won't actually stay radioactive unless it's contaminated with radioactive material.

**Comment by RemieRichards** [comment]
@Anonmare it uses CHECK_TICK, which pauses the system's processing if the game gets too laggy, it's the thing MSO added based on the new byond update, the thing that let us kill lag 5ever

**Comment by xxalpha** [comment]
Here's how radiation doesn't work: rad pulses.

**Comment by Thunder12345** [comment]
It's still a better approximation than every single thing ever exposed to radiation magically becoming radioactive itself. Just make the pulses weaker and happen more often.

**Comment by xxalpha** [comment]
It's not a better approximation. I'll show you a better approximation.

**Comment by duncathan** [comment]
This is not gonna be frozen; the turf changes are small enough to deal with

No comment on my opinion of the pr

**Comment by Delimusca** [comment]
@Thunder12345 i don't know much about the radiation emissions of black holes, but i know that high energy particles colliding with atoms can knock off baryons, creating a possibly unstable isotope that will later decay.
that's how nuclear fallout works, and i don't see why it would be unrealistic unless something about black holes prevents that.

**Comment by xxalpha** [comment]
Fallout is residue, which is not the same as induced radiation. Don't worry, I am changing this to have both induced/fallout and normal radiation.

**Comment by Razharas** [comment]
>Linear induced radiation dissipation

I seriously hope you guys dont do this

**Comment by Iamgoofball** [comment]
>>>green
>>>red

>>>NOT BLUE

**Comment by Bawhoppen** [comment]
:-1: it doesn't fit in this game

**Comment by xxalpha** [comment]
Neither do you.

**Comment by optimumtact** [comment]
I like it

**Comment by KazeEspada** [comment]
Bonus points if you make space lukewarm and rad you to death instead.

**Comment by xxalpha** [comment]
I could make space inherently radioactive, but that would probably be bad for balance.

**Comment by Bawhoppen** [comment]
advanced future space suits block out the space radiation

**Comment by AnturK** [comment]
I like it, needs to dissipate slow naturally and get some decontamination gear though. Also change the radiation event.

**Comment by xxalpha** [comment]
It decontaminates slowly. A turf with max radiation (500) takes 40 minutes to go back to zero.

Tools to clean up radiation should come in the future, of course.

**Comment by Core0verload** [comment]
+ "controversial" tag

**Comment by duncathan** [comment]
It seems few people like this

**Comment by xxalpha** [comment]
Few people liked goofchem.

**Comment by Iamgoofball** [comment]
this is actually a good idea and I will reopen and rip off this PR if it doesnt get reopened @duncathan

**Comment by duncathan** [comment]
convince another maintainer to reopen it

**Pull request closed without merging**